storage/path.go

```diff
@@ -125,7 +125,7 @@ func (p Path) Ref(head *ast.Term) (ref ast.Ref) {
 	ref = make(ast.Ref, len(p)+1)
 	ref[0] = head
 	for i := range p {
-		idx, err := strconv.ParseInt(p[i], 10, 64)
+		idx, err := strconv.ParseInt(p[i], 10, 32)
```

Comment:
Alternatively we could `strconv.ParseUInt` and then call `ast.UIntNumberTerm` below. That way we can index into arrays larger than 2**32? Not sure

Reply:
Hmm good idea. I'll try that.

Reply:
....in a follow-up 😎